x-pack/plugins/security_solution/public/detections/components/rules/eql_query_bar/eql_query_bar.test.tsx

```diff
@@ -25,15 +25,27 @@ describe('EqlQueryBar', () => {
   });
 
   it('renders correctly', () => {
```

Comment:
should we add  test that ensures FilterBar is rendered?

Reply:
might be worth consider to add a Cypress test x-pack/plugins/security_solution/cypress/integration/detection_rules/event_correlation_rule.spec.ts